config.go

```diff
@@ -60,6 +61,10 @@ func newConfiguration() *configuration {
 // Config contains library level configuration
 var Config *configuration
 
+const (
+	BleveVectorSearch = iota
```

Comment:
I understand what you're trying to do here, but could you look into an other approach that doesn't change the config construct. This is applicable for testing only and the scope of the change must be just that.

Reply:
a global variable in bleve package ?
searchKnn.go -> const BleveVectorSearchFeature = true
searchNoKnn.go -> const BleveVectorSearchFeature = false

Reply:
Ya sounds better.